es/ponyfill.js

```diff
@@ -0,0 +1,17 @@
+export default function symbolObservablePonyfill(root) {
+	let result;
```

Comment:
So, the weird thing about ES build is it shouldn't use any other features except modules. The point is not use ES6, it's to use the new module system so Rollup can do its thing.

In Redux, we actually have two build envs. I toggle them with BABEL_ENV and compile src to lib and es. Both are compiled with Babel, but one of them is missing a CommonJS output plugin so the ES module syntax stays preserved. But everything else is compiled. 

Does this make sense? I think .babelrc and package.json in Redux should give an idea. You could use es2015-rollup preset instead of listing plugins explicitly, and add two CommonJS plugins to your CommonJS env override (basic support and that other default export plugin).

One last thing: you'd need to point to ES build as jsnext:main in package.json.